changelogs/head.asciidoc

```diff
@@ -9,6 +9,7 @@ https://github.com/elastic/apm-server/compare/8.6\...main[View commits]
 - `transaction.success_count` has been moved to `event.success_count` {pull}9819[9819]
 - Stop indexing transaction metrics to `metrics-apm.internal` {pull}9846[9846]
 - Stop indexing span destination metrics to `metrics-apm.internal` {pull}9926[9926]
+- Service metrics and service summary aggregations are now always enabled {pull}10060[10060]
```

Comment:
@simitt here I'm calling this a breaking change since similar changes above e.g. "Stop indexing transaction metrics to `metrics-apm.internal`" are also called breaking. Can you please confirm if this applies to this PR as well? Or should this be under another section of the changelog?

Reply:
Let's add this as enhancement - we are adding new functionality, not removing existing one. The feature was disabled by default since it was in `beta`.